sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -259,4 +268,72 @@ export class InstitutionController extends BaseController {
       body.isActive,
     );
   }
+
+  @Get("/my-details")
+  async getMyDetails(
+    @UserToken() token: IInstitutionUserToken,
+  ): Promise<InstitutionUserAndAuthDetailsDto> {
+    // Get logged in user and location details with auth
+    const institutionUser =
+      await this.institutionService.getInstitutionUserByUserName(
```

Comment:
It seems that there is nothing being used from the institution data.
Please consider retrieving only the user from DB.